### PR TITLE
Move test fixture files

### PR DESCRIPTION
**Pull request by arbulu89**
Move fixture test files to the fixtures folder, to more meaningful places.
Besides that, I have changes the way of accessing the test files. The old `testing.go` (used with the `_ "github.com/trento-project/agent/test"`) import was creating some confusion as it changes the execution directory for all of the tests in the same package.
The new way is more explicit.

### CODE REVIEW

File: test/helpers/fixtures.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package helpers
+
+import (
+	"path"
+	"runtime"
+)
+
+var fixturesFolder = "" // nolint
```
Comment (dottorblaster) [inline test/helpers/fixtures.go:8]: Why this nolint?

**Review by dottorblaster — COMMENTED** [review]
Just a tiny comment

File: test/helpers/fixtures.go (hunk 1 above)
Comment (arbulu89) [reply test/helpers/fixtures.go:8]: Because `golangci-lint` doesn't like global variables, but this is a corner case. I could put `// nolint:gochecknoglobals` if that helps.

Comment (arbulu89) [reply test/helpers/fixtures.go]: Changed and rebased

**Review by nelsonkopliku — APPROVED** [review]
Looks good.

**Review by CDimonaco — COMMENTED** [review]
Just a note, the name of the function is GetFixtureFile, the name could be misleading because I expect that the function will actually return the file, could we call it GetFixturePath, or embed di os open in that and return the bytes/string

**Comment by arbulu89** [comment]
> Just a note, the name of the function is GetFixtureFile, the name could be misleading because I expect that the function will actually return the file, could we call it GetFixturePath, or embed di os open in that and return the bytes/string?
> 
> What do you think?

Yes, you are right. Better name it `GetFixturePath`.
Thanks!

**Review by CDimonaco — APPROVED** [review]
LGTM
